dvc/api.py

```diff
@@ -92,6 +94,160 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
         return fd.read()
 
 
+def params_show(
```

Comment:
Wait, why? It’s a getter. Again, I am confused with porcelain/API discussion.

Reply:
@skshetry The argument during the last retro discussion was that this is at least following the CLI conventions. Could consider adding `get_params` as an alias though.

Reply:
Right, but is `dvc.api` the place to have these CLI wrappers?

Reply:
> CLI wrappers?

What do you call CLI wrappers? I think the idea is that we follow CLI *conventions* regarding names, but this doesn't mean that the API is a CLI wrapper. 
This API uses the internal Python API, raises exceptions, and returns a structure meant to be used in Python (instead of the internal structure returned by CLI `--json` options, for example).

Reply:
Personally, my vote would be `get_params` because the most immediate use case here is to onboard new users who aren't already familiar with the CLI.

However, I would rather get this merged than continue to discuss naming.